2DIterativeRecon.jl

```diff
@@ -0,0 +1,431 @@
+## Matlab Author: Rodrigo de Barros Vimieiro
+## Translated by Sonia Minseo Kim
+
+## 2-D Branchless Distance Driven Code
+using Plots: plot
+using ImagePhantoms: shepp_logan, SheppLoganToft
+using ImageGeoms: ImageGeom, axesf
+using MIRTjim: jim, prompt #specify the functions you are using in the packages
+using LazyGrids: ndgrid 
+using Dierckx: Spline1D
+
+# Julia is compiling program so you need to restart the program in order to clear the variables 
+
+## Geometry Definitions
+# Geometry
+deg = 1
+geo = (DSD = 100000, DSO = 99700, pSize = 1, dSize = 0.5, nPix = 256, nDet = 1024
+        , theta = deg2rad.(0:deg:360-deg), isoX = 0, isoY = 0) #Define tuple
+
+# Make the Phantom Image
+phantomImg = shepp_logan(geo.nPix, SheppLoganToft())
+
+## Projection Branchless
+function projectionBranchless(phantom,geo ; draw::Bool = false)
+
+DSD = geo.DSD      
+DSO = geo.DSO      
+pSize = geo.pSize    
+dSize = geo.dSize    
+nPix = geo.nPix      
+nDet = geo.nDet     
+theta = geo.theta
+
+# Detector boundaries
+detX = (-(nDet/2):(nDet/2)) .* dSize
+detY = (-(DSD-DSO)-(dSize/2)) .* ones(nDet+1)
+
+# Pixel boundaries
+(pixelX,pixelY) = ndgrid(-(nPix/2):(nPix/2), -(nPix/2):(nPix/2))
+pixelX = pixelX .* pSize
+pixelY = reverse(pixelY .- pSize/2, dims = 1)
+
+# Tube
+tubeX = 0     
+tubeY = DSO    
+
+# Iso-center
+isoX = geo.isoX
+isoY = geo.isoY
+
+sinogram = zeros(size(theta,1),nDet)
```

Comment:
```suggestion
sinogram = zeros(T, nDet, length(theta))
```

eventually i prefer this way, after it all works...